src/java/org/apache/cassandra/index/sai/disk/v1/trie/TrieTermsDictionaryReader.java

```diff
@@ -95,6 +95,38 @@ public long exactMatch(ByteComparable key)
         return getCurrentPayload();
     }
 
+    public long ceiling(ByteComparable key)
```

Comment:
Could we add some javadoc to this and `floor` just giving the expectations of the method. It would be useful when we come back to it.

Reply:
Yes, I'll definitely add javadocs.